packages/server-ct/src/routes-ct.ts

```diff
@@ -22,6 +22,7 @@ export interface InitializeRoutes {
   networkProxy: NetworkProxy
   getRemoteState: () => any
   onError: (...args: unknown[]) => any
+  testingType: 'component' | 'e2e'
```

Comment:
Can grab the type here to avoid duplication: https://github.com/cypress-io/cypress/blob/bf6560691436a5a953f7e03e0ea3de38f3d2a632/cli/types/cypress.d.ts#L11

Reply:
`Cypress['TestingType']`